.github/workflows/build-bootc.yaml

```diff
@@ -1,11 +1,9 @@
 name: "Build Bootc Agent Bootstrap Images"
 on:
-  pull_request_target:
-  schedule:
-    - cron: '0 */12 * * *'
```

Comment:
So before this would run the image workflow whenever a PR was opened, AND on a schedule (every 12 hours).

With your changes this only get's run when manually triggered 

I would think the logic we really want is, 
- Run when manually triggered
OR
- Run whenever a new commit is merged into Main

And for bonus points we could run the build but NOT push step on PRs which change either the build-bootc.yaml or associated dockerfiles to ensure they aren't broken by a PR 

WDYT @oglok?

Reply:
Yeah, so there is already two pipelines. One for Fedora, and one for Centos. These are scheduled every 12 hours (since this repo does not change a lot) + every PR.

I want a new pipeline, based on RHEL that can be manually triggered (for the moment).